src/App.js

```diff
@@ -10,7 +10,9 @@ import Watchlater from './pages/watchlater/Watchlater'
 import Login from './pages/auth/login/Login'
 import Signup from './pages/auth/signup/Signup'
 import Mockman from "mockman-js";
-
+import Singlevideo from "./pages/singlevideo/Singlevideo";
+import SinglePlaylist from "./pages/singleplaylist/SinglePlaylist";
```

Comment:
Create a single file in src folder and try to use barell imports.

Reply:
Yet to learn how to do this, will do this in a future refactoring run.